database/migrations/2021_05_31_125925_add_hash_to_images_table.php

```diff
@@ -14,7 +14,7 @@ class AddHashToImagesTable extends Migration
     public function up()
     {
         Schema::table('images', function (Blueprint $table) {
-            $table->string('hash')->nullable();
+            $table->longText('hash')->nullable();
```

Comment:
You can use the second argument of `string()` to specify a string length (defaults to 100). As the hash length should be fixed, this is probably more memory efficient.
```suggestion
            $table->string('hash', 1024)->nullable();
```